### DISCUSSION TITLE
How do I update antd

### DISCUSSION THREAD

**Discussion by manivsh** [ant-design/ant-design#43652] [Q&A] [answered]
I had an issue (https://github.com/ant-design/ant-design/issues/43202), which was resolved with a merge in (https://github.com/react-component/trigger).

So my question is: How do I update my local project which depend on antd?

I've tried `npm i antd@latest`, which updates antd but the bug persist.
I've also tried deleting `/node_module` and then `npm cache clean --force`, but got no changes.

What am I missing?

**Comment by afc163** [comment, marked as answer]
`rm package-lock.json yarn.lock && npm install`
